app/views/events/participants.html.erb

```diff
@@ -1,25 +1,16 @@
 <%- model_class = Event -%>
 <div class="page-header">
-  <h1><%=t '.title', :default => model_class.model_name.human.titleize %></h1>
+  <h1><%=t '.title', :default => model_class.model_name.human.titleize %> (<%= @event.kind.humanize %>)</h1>
 </div>
 
-<dl class="dl-horizontal">
-  <dt><strong><%= model_class.human_attribute_name(:name) %>:</strong></dt>
-  <dd><%= @event.name %></dd>
-  <dt><strong><%= model_class.human_attribute_name(:description) %>:</strong></dt>
-  <dd><%= @event.description %></dd>
-  <dt><strong><%= model_class.human_attribute_name(:max_participants) %>:</strong></dt>
-  <dd><%= @event.max_participants %></dd>
-  <dt><strong><%= model_class.human_attribute_name(:active) %>:</strong></dt>
-  <dd><%= @event.active %></dd>
-</dl>
+<%= render :partial => 'show_event' %>
 
-<h1>Teilnehmer</h1>
+<h1><%= t '.participants' %></h1>
 <table>
```

Comment:
You could take a look at some Bootstrap classes for tables, it makes them look a little less depressing ;)
http://www.w3schools.com/bootstrap/bootstrap_tables.asp